src/search/_base.scss

```diff
@@ -49,4 +49,63 @@
 			margin-right: auto;
 		}
 	}
+
```

Comment:
The following CSS need to be scoped by the page type CSS ```.page-type-search```.
FYI - I am going to do that update and I am working on it to help you to finalize this search template

Reply:
Thank you Pierre!